src/components/messenger/list/index.tsx

```diff
@@ -241,12 +239,14 @@ export class Container extends React.Component<Properties, State> {
 
         {this.props.stage === SagaStage.None && (
           <div {...cnMessageList('profile-bar')}>
-            {this.props.includeRewardsAvatar && (
+            {this.props.isMessengerFullScreen && (
               <div {...cnMessageList('avatar-container')}>{this.renderSettingsMenu()}</div>
             )}
 
             <FeatureFlag featureFlag='enableRewards'>
-              <div {...cnMessageList('rewards-container', this.props.includeRewardsAvatar && 'with-avatar')}>
+              <div
+                {...cnMessageList('rewards-container', this.props.isMessengerFullScreen && 'with-full-screen-modifier')}
```

Comment:
how do we feel about `with-full-screen-modifier` ? I felt like this made more sense but if there are better suggestions please let me know 🙂